### ISSUE TITLE
pylint wrongly complains about no-member error in levenshtein module

### ISSUE THREAD

**Issue by martimlobao** [PyCQA/pylint#3369]
### Steps to reproduce
1. `pip install python-Levenshtein`
2. 
```python
# leventest.py
import Levenshtein

Levenshtein.distance("apple", "aple")
```
3. `pylint leventest.py`

### Current behavior
```
$ pylint leventest.py 
************* Module research.leventest
leventest.py:3:0: E1101: Module 'Levenshtein' has no 'distance' member (no-member)
```

### Expected behavior
`pylint` should not incorrectly complain about missing member `distance`.

### pylint --version output
```
pylint 2.4.4
astroid 2.3.3
Python 3.6.6 (default, Feb 12 2019, 20:41:17) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.10.44.4)]
```

**Comment by PCManticore** [comment]
Thanks for the report.

`Levenshtein` is a C module and `pylint` can't help much with these. You have a couple of choices to get rid of this error:

* pass `--extension-pkg-whitelist=Levenshtein` to pylint to force it to import the module to build an AST from it.
* pass `--generated-members=distance` to fake the existence of that attribute so `pylint` does not emit the error
* pass `--ignored-modules=Levenshtein` to skip this module altogether.
* disable the error in place where it occurs

Hope this helps.
